hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -6,12 +6,33 @@
 class CRM_Hrjobcontract_Upgrader extends CRM_Hrjobcontract_Upgrader_Base {
 
   public function install() {
-
     // $this->executeCustomDataFile('xml/customdata.xml');
     $this->executeSqlFile('sql/install.sql');
     $this->upgradeBundle();
   }
 
+  public function enable() {
```

Comment:
Removed the methods on Upgrader class and tested enbling and disabling extension.  Worked ok!